src/utils/leads.js

```diff
@@ -12,10 +12,8 @@ export const save_form = async (formData=null, tags=[], automations=[], session=
     if(typeof(formData) !== 'object') throw Error("Missing formData");
     const getEnvironmentAPI = () => {
         if (process.env.NODE_ENV === 'development') {
-            // development case
             return process.env.GATSBY_BREATHECODE_TEST
```

Comment:
Now cypress can run server and start the test in development (with the dependency `start-server-and-test`)